addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -57,7 +57,7 @@ export var gravity := -9.8
 export var push_rigid_bodies := true
 
 ## GroundPhysicsSettings to apply - can only be typed in Godot 4+
-export (Resource) var physics = null setget set_physics
+export var physics: Resource = null setget set_physics
```

Comment:
I didn't know this notation already worked in Godot 3, cool :)

Reply:
I just re-checked and it does select the resource picker in Godot 3.4.5. This is one where it'll work in 4, but we'll still probably want to upgrade it to the typed-resource anyway.